application.py

```diff
@@ -0,0 +1,207 @@
+import csv
+import datetime
+import re
+
+import matplotlib.pyplot as plt
+
+"""
+    Initialized categories list to choose correct option for add new spending.
+    todo:: add some more categories, create custom categories
+    todo:: (add them to other file and read to list to manipulate with them!)
+"""
+
+categoriesList = ["Food", "Clothes", "Entertainment", "Transportation", "Other"]
+
+
+def parseCategorieToNumber(category: str) -> int:
+    for i in range(len(categoriesList)):
+        if categoriesList[i] == category:
+            return i + 1
+    return -1
+
+
+class spending:
```

Comment:
According to PEP-8 class names must be written in CamelCase: "Spending"

Lets also use single class per module